libraries/AP_RangeFinder/AP_RangeFinder_Backend_CAN.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_RangeFinder_Backend_CAN.h"
+
+#if HAL_MAX_CAN_PROTOCOL_DRIVERS
+
+const AP_Param::GroupInfo AP_RangeFinder_Backend_CAN::var_info[] = {
+
+    // @Param: RECV_ID
+    // @DisplayName: CAN receive ID
+    // @Description: The receive ID of the CAN frames. A value of zero means all IDs are accepted.
+    // @Range: 0 65535
+    // @User: Advanced
+    AP_GROUPINFO("RECV_ID", 13, AP_RangeFinder_Backend_CAN, receive_id, 0),
+
+    // @Param: SNR_MIN
+    // @DisplayName: Minimum signal strength
```

Comment:
It would be nice to have this Display name start with "RangeFinder" (or similar)